system/xmllib/libxml_xpathprocessor.cpp

```diff
@@ -991,8 +991,24 @@ class CLibXpathContext : public CInterfaceOf<IXpathContext>, implements IVariabl
         switch (obj->type)
         {
             case XPATH_NODESET:
-                toXMLNodeSet(obj, content);
-                isValue = false;
+                if (isValue)
```

Comment:
I'm not sure if it's a big deal, but I'm not a huge fan of using the same parameter as an input that may or may not be modified to become an output.  It can get a bit confusing.